geoportailv3/views/luxprintproxy.py

```diff
@@ -0,0 +1,149 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2011-2015, Camptocamp SA
+# All rights reserved.
+
+# Redistribution and use in source and
+# binary forms, with or without
+# modification, are permitted provided
+# that the following conditions are met:
+
+# 1.    Redistributions of source code must
+#       retain the above copyright notice, this
+#       list of conditions and the following disclaimer.
+# 2.    Redistributions in binary form mustg
+#       reproduce the above copyright notice,
+#       this list of conditions and theg
+#       following disclaimer in the documentation
+#       and/or other materials provided
+#       with the distribution.
+
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT
+# HOLDERS AND CONTRIBUTORS "AS IS" AND
+# ANY EXPRESS OR IMPLIED WARRANTIES,
+# INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND
+# FITNESS FOR A PARTICULAR PURPOSE ARE
+# DISCLAIMED. IN NO EVENT SHALL THE
+# COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+# ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+# (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT
+# OF SUBSTITUTE GOODS OR SERVICES;
+# LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND
+# ON ANY THEORY OF LIABILITY, WHETHER IN
+# CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE)
+# ARISING IN ANY WAY OUT OF THE USE OF THIS
+# SOFTWARE, EVEN IF ADVISED OF THE
+# POSSIBILITY OF SUCH DAMAGE.
+
+# The views and conclusions contained in the
+# software and documentation are those
+# of the authors and should not be interpreted
+# as representing official policies,
+# either expressed or implied, of the FreeBSD Project.
+
+import json
+import logging
+from cStringIO import StringIO
+
+from pyramid.view import view_config
+from pyramid.response import Response
+
+from PyPDF2 import PdfFileMerger
+import weasyprint
+
+from c2cgeoportal.views.printproxy import PrintProxy
+from c2cgeoportal.lib.caching import init_cache_control, get_region
+
+log = logging.getLogger(__name__)
+cache_region = get_region()
+
+
+class LuxPrintProxy(PrintProxy):
+
+    ref_spec = {}
```

Comment:
> Which is the case already?

No in the print I don't see anything to externalise it, but originally it was designed for this ...